src/uwtools/drivers/forecast.py

```diff
@@ -57,30 +51,32 @@ def create_model_config(self, config_obj, outconfig_file, base_file=None):
         will need to be made for the other apps.
 
         Args:
-            config_obj: in-memory dictionary initialized by object.
-                        values override any settings in base file
-            outconfig_file: location of output configuration file
-            base_file: optional path to file to use as a base file
-        '''
+            config_file: Path to user configuration file
+            outconfig_file: Path to output configuration file
+            base_file: Path to optional file with base configuration
+        """
+
+        # #PM# DEAL WITH ISSUE OF config_file / config_obj TYPE.
```

Comment:
I think it's fine for now. We do have work on the backlog to make sure that this one comes together in a working fashion. It would be good to have a review of what "drivers" are meant to accomplish before, or as part of, that work. We can decide what we need (config obj or config file) at which points in the process of the driver.